apps/emqx_gateway/src/lwm2m/lwm2m_xml/LWM2M_Server-v1_0.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why are these files changed ?

Reply:
I run `find . -type f -print0 | xargs -0 dos2unix` to convert dos to unix format.
the command causes the change.

if I run `make clean-all & make ` on my mac machine will cause a lot files report:
`Bash script and /bin/bash^M: bad interpreter: No such file or directory`.
There are some files that are c library dependent.
so I run dos2unix command to covert this.